sql/index/pilosa/lookup.go

```diff
@@ -63,6 +64,7 @@ type (
 		keys        []interface{}
 		expressions []string
 		operations  []*lookupOperation
+		indexes     []string
```

Comment:
Maybe it should be `map[string]int`, where key is an index name and value will be order id.
So we'll have auto-uniques (because of map), and then we can sort map entries by value, sth. like:
```go
m := make(map[string]int)
m[indexName] = len(m)
//...
indexes := make([]string, len(m))
for k, v := range m {
    index[k] = v
}
return index
```

WDYT?

Reply:
Why the `int`? We can just use a `map[string]struct{}` and then sort them.

Also, even if `indexes` is a map, it may not be unique, because `id` can be inside that map.

Reply:
The question is how do we want to show indexes (sorted in the order how filter was parsed, or sorted by name)?
On the other hand, these are "small" numbers so most likely keep the most readable version.

Reply:
I will use the `struct{}` only because the int requires some explanation at first sight and the other doesn't. Regarding the order, I think it does not matter much, so we can just sort.

Reply:
Ok, so let's keep it what we have so far. I suppose, there is no many cases where we'll use the same index more than once, so maybe map can be kind of overhead.